src/python/utils/index_storage.py

```diff
@@ -14,6 +14,7 @@ class IndexStorage(object):
 
     def __init__(self):
         self._index_storage = boto3.client('s3')
+        self._active_partitions = ([]);
```

Comment:
What is the intention with this field? It doesn't look like get_active_partitions() is called anywhere, is this field necessary?